Automattic/wp-calypso
# Typography: NPM package to add support for Recoleta

#### Changes proposed in this Pull Request

* This package includes Sass and CSS files that import Recoleta and set basic selectors to determine when to show the font across different languages.

Still to do:

* We need to determine which languages Recoleta best supports and include them in the language selector.

**Visuals**

<img width="961" alt="Screen Shot 2020-06-09 at 8 12 25 AM" src="https://user-images.githubusercontent.com/2124984/84146089-1ae7c780-aa29-11ea-802b-2eb22a7d015f.png">

<img width="1086" alt="Screen Shot 2020-06-09 at 8 12 53 AM" src="https://user-images.githubusercontent.com/2124984/84146097-1e7b4e80-aa29-11ea-8986-7bb8fad586f6.png">

#### Testing instructions

* Switch to this PR and `yarn start`
* Add the following to `client/assets/stylesheets/shared/_typography.scss` (or anywhere you want to use the class name): `@import '~@automattic/typography/sass/fonts';`
* Add `wp-brand-font` as a class name to any element on which you want to test the font
* Recoleta should show on the element you've chosen if you're using one of the supported languages (Latin-based languages, mostly); if not, you should see the fallback serif.
